contracts/modules/maker/MakerV2Base.sol

```diff
@@ -0,0 +1,100 @@
+pragma solidity ^0.5.4;
```

Comment:
Is this missing a `Copyright (C) 2018  Argent Labs` at the top?

Reply:
When we add the copyright we should set the year to 2019.

Reply:
Done